pkg/kritis/admission/admission.go

```diff
@@ -225,6 +236,20 @@ func reviewPod(pod *v1.Pod, ar *v1beta1.AdmissionReview) {
 	reviewImages(pods.Images(*pod), pod.Namespace, pod, ar)
 }
 
+func reviewReplicaSet(replicaSet *appsv1.ReplicaSet, ar *v1beta1.AdmissionReview) {
+	// First, check for a breakglass annotation on the replica set
+	if checkBreakglass(&replicaSet.ObjectMeta) {
+		glog.Infof("found breakglass annotation for %s, returning successful status", replicaSet.Name)
+		return
+	}
+	for _, c := range replicaSet.Spec.Template.Spec.InitContainers {
```

Comment:
How about we collect all images together. That ways, the reviewImages will prune duplicate images.
Calling them one by one would not do that.

Reply:
Can we add unit tests for this?

Reply:
Done!